inst/www/flex_dashboard/flexdashboard.js

```diff
@@ -299,6 +297,12 @@ var FlexDashboard = (function () {
       container.append(li);
     }
 
+    // mark active tab and corresponding nav menu item
+    if (active) {
+      wrapper.addClass('active');
+      li.addClass('active');
+    }
```

Comment:
Apologies for just realizing this, but this feels slightly cleaner/better?

```suggestion
    if (active)
      $(a).tab("show");
```